js/index.js

```diff
@@ -1,6 +1,36 @@
-// Iteration 1: Names and Input
+// console.log("I'm ready!");
+let hacker1 = "Helena";
+console.log("The driver's name is " + hacker1);
 
-// Iteration 2: Conditionals
+let hacker2 = "Denise";
+console.log("The navigator's name is " + hacker2);
 
 
-// Iteration 3: Loops
+// iteration 2: conditionals
+if (hacker1.length > hacker2.length) {
+  console.log(`The driver has the longest name, it has ${hacker1.length} characters`); 
+} else if (hacker1.length < hacker2.length) {
+  console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters`);  
+} else if (hacker1.length == hacker2.length) {
+  console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`);
+}
```

Comment:
good job, just so you know: you could also write if, else if, else like this:
...
else {
  console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`);
}